packages/form/src/form/VirtualizedAutocompleteList.tsx/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const LISTBOX_PADDING = 8
+export const CHARACTERS_PER_LINE = 25
+export const MAX_VISIBLE_ITEM_COUNT = 8
```

Comment:
Folder name should not contain .tsx 
packages/form/src/form/VirtualizedAutocompleteList**.tsx**/constants.ts